packages/editor/src/components/parts/permissions/PermissionsPart.tsx

```diff
@@ -0,0 +1,28 @@
+import { PartProps, usePartDirty, usePartState } from '../../../components/editors';
+import { PathContext, useValidations } from '../../../context';
+import { Checkbox } from '../../../components/widgets';
+import { usePermissionsData } from './usePermissionsData';
+import { PermissionsData } from '@axonivy/inscription-protocol';
+
+export function usePermissionsPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = usePermissionsData();
+  const compareData = (data: PermissionsData) => [data];
+  const validation = useValidations(['permissions']);
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);

```

Comment:
this part has no validations, so you can simply insert an empty array into usePartState